app/controllers/epp/sessions_controller.rb

```diff
@@ -13,6 +13,9 @@ def login
       webclient_request = ENV['webclient_ips'].split(',').map(&:strip).include?(request.ip)
       if webclient_request && !Rails.env.test? && !Rails.env.development?
         client_md5 = Certificate.parse_md_from_string(request.env['HTTP_SSL_CLIENT_CERT'])
+        if ENV['cert_path'].nil?
+          logger.error('webclient cert (cert_path) missing, registrar epp/repp disabled')
```

Comment:
I don't see a point continuing execution if there is no `cert_path` configured in this case, therefore `fail 'message'` might be better here.

Reply:
Could you give me example don't exactly understand what you mean

Reply:
I was thinking about something like `fail 'webclient cert (cert_path) missing'` unless `ENV['cert_path']`.present?`.

Reply:
style guide seems to prefer raise over fail